libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,359 @@
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: var(--spacing-xxs);
+  margin-top: 0;
+  color: var(--color-black);
+}
+
+.merch-card .consonant-SpecialOffers-inner,
+.merch-card .consonant-PlansCard-inner {
+  justify-content: center;
+}
+
+.merch-card.special-offers .consonant-SpecialOffers-description p,
+.merch-card.special-offers .consonant-PlansCard-description p {
+  font-size: var(--type-body-xs-size);
+}
+
+.merch-card.special-offers .consonant-CardFooter {
+  margin-top: 0;
+  padding-top: 0;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.merch-card.special-offers h4 {
+  font-size: var(--type-body-xxs-size);
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+
+.merch-card .consonant-CardFooter .standard-wrapper,
+.merch-card .consonant-CardFooter .checkbox-container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container {
+  justify-content: flex-start;
+  margin-bottom: var(--spacing-s);
+  cursor: pointer;
+}
+
+.merch-card .consonant-CardFooter .standard-wrapper {
+  height: 32px;
+  white-space: nowrap;
+}
+
+.merch-card .secure-transaction-wrapper {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+  flex-shrink: 1;
+}
+
+.merch-card .secure-transaction-label {
+  font-size: var(--type-body-xxs-size);
+  color: var(--color-gray-700);
+}
+
+.merch-card .secure-transaction-icon {
+  display: inline-block;
+}
+
+.merch-card .secure-transaction-icon-img {
+  display: block;
+  height: 15px;
+  width: 12px;
+}
+
+
+.merch-card .consonant-CardFooter .checkbox-container .checkbox-label {
+  font-size: var(--type-body-xxs-size);
+  color: var(--color-gray-700);
+  white-space: nowrap;
+}
+
+.merch-card .secure-transaction-label,
+.merch-card .consonant-CardFooter .checkbox-container .checkbox-label {
+  flex-grow: 1;
+  margin-left: 5px;
+}
+
+.merch-card .secure-transaction-icon,
+.merch-card .checkbox-container .checkmark {
+  flex-shrink: 0;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container input[type="checkbox"] {
+  display: none;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container .checkmark {
+  position: relative;
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  border: 2px solid #757575;
+  background: #fff;
+  border-radius: 2px;
+  cursor: pointer;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container input[type="checkbox"]:checked + .checkmark {
+  background-color: var(--color-accent);
+  background-image: url('./img/check-mark-small.svg');
+  border-color: var(--color-accent);
+}
+
+@media only screen and (min-width: 600px) {
+  .consonant-CardsGrid--2up {
+    grid-template-columns: repeat(auto-fit, minmax(380px, max-content)) !important;
+  }
+
+  .consonant-CardsGrid--4up {
+    grid-template-columns: repeat(auto-fit, minmax(260px, max-content)) !important;
+  }
+}
+
+@media only screen and (max-width: 1200px) {
+  .merch-card.segment-blade {
```

Comment:
These are causing some strange alignments in [my draft comparing](https://merch-card--milo--adobecom.hlx.page/drafts/rparrish/cards/merch-cards) section grid sizes to card grid sizes. 
Are these values part of the requirements or should we just be using the grid defaults?

<img width="1083" alt="Screen Shot 2023-08-22 at 4 54 44 PM" src="https://github.com/adobecom/milo/assets/2086146/3a649e1f-83e7-4342-8203-c9376f867f96">